src/app/pushclient.js

```diff
@@ -1,86 +1,129 @@
-var request = require('request');
+/**
+ * You need to rename this file to PushClient.js
+ * Good OS are case-sensitive.
+ */
+
+const request = require('request');
 
 module.exports = class PushClient {
 
   constructor(pushConfig, retryOptions) {
-    if(!pushConfig.organizationId) throw new Error(`PushClient needs an organizationId in the pushConfig.`);
-    if(!pushConfig.sourceId) throw new Error(`PushClient needs an sourceId in the pushConfig.`);
-    if(!pushConfig.pushtoken) throw new Error(`PushClient needs an pushtoken in the pushConfig.`);
+    if(!pushConfig.organizationId) {
+      throw new Error(`PushClient needs an organizationId in the pushConfig.`);
+    }
+    if(!pushConfig.sourceId) {
+      throw new Error(`PushClient needs an sourceId in the pushConfig.`);
+    }
+    if(!pushConfig.pushApiKey) {
+      throw new Error(`PushClient needs an pushApiKey in the pushConfig.`);
+    }
     this.organizationId = pushConfig.organizationId;
     this.sourceId = pushConfig.sourceId;
-    this.pushtoken = pushConfig.pushtoken;
+    this.pushApiKey = pushConfig.pushApiKey;
```

Comment:
I'm more of a fan of Google's JavaScript Style guide:
https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined